### PR TITLE
Add the new Product drop-down with the Floating menu

### PR DESCRIPTION
**Pull request by alexventuraio**
### This is how it scales on screen
product-dropdown-menu

### CODE REVIEW

**Comment by alexventuraio** [comment]
[Public | Update devpost.com top level navigation | Product dropdown](https://trello.com/c/PK73cvEu/8336-public-update-devpostcom-top-level-navigation-product-dropdown)

File: app/views/reimagine2/devpost/global_nav/_world_menu_entries.html.erb [hunk 1]
```diff
@@ -1,4 +1,42 @@
...
+        <div class="panel-container">
+          <a class="panel-product panel-product-content" href="/product/public-hackathons">
+            <%= image_tag 'reimagine2/devpost-logo-mobile.svg', alt: 'Devpost' %>
+            <div>
+              <p class="title bold">Devpost</p>
+              <p class="subtitle">Grow your developer ecosystem and promote your platform.</p>
+            </div>
+          </a>
+          <div class="panel-content">
+            <a href="#">Hackathons</a>
```
Comment (devpost-mzheng) [inline app/views/reimagine2/devpost/global_nav/_world_menu_entries.html.erb:17]: what about the URLs for these?

File: app/views/reimagine2/devpost/global_nav/_world_menu_entries.html.erb [hunk 2]
```diff
@@ -1,4 +1,42 @@
...
+        <div class="panel-container">
+          <a class="panel-product panel-product-content" href="/product/devpost-for-teams">
+            <%= image_tag 'reimagine2/dft-logo-mobile.svg', alt: 'Devpost for Teams' %>
+            <div>
+              <p class="title bold">Devpost for Teams</p>
+              <p class="subtitle">Drive innovation, collaboration, and retention within your organization.</p>
+            </div>
+          </a>
+          <div class="panel-content">
+            <a href="#">Teams login</a>
```
Comment (devpost-mzheng) [inline app/views/reimagine2/devpost/global_nav/_world_menu_entries.html.erb:32]: and these?

**Comment by devpost-mzheng** [comment]
it looks like URLs are missing. Is it possible to post some visuals too? as like a video so we can see how it scales?

**Comment by alexventuraio** [comment]
> it looks like URLs are missing. Is it possible to post some visuals too? as like a video so we can see how it scales?

Thanks @devpost-mzheng , I have updated the PR with a gif and set the proper URLs on the different links.

File: app/views/reimagine2/devpost/global_nav/mobile/_world_menu_entries.html.erb [hunk 3]
```diff
@@ -1,4 +1,27 @@
 <li class="divider"></li>
+<li class="mobile-product-menu panel-container">
+  <a class="panel-product panel-product-content" href="https://info.devpost.com/product/public-hackathons">
+    <%= image_tag 'reimagine2/devpost-logo-mobile.svg', alt: 'Devpost' %>
+    <p class="title bold">Devpost</p>
+  </a>
+  <div class="panel-content">
+    <a href="https://devpost.com/hackathons">Hackathons</a>
```
Comment (devpost-mzheng) [inline app/views/reimagine2/devpost/global_nav/mobile/_world_menu_entries.html.erb:8]: `<li class="divider"></li>
<li>
  <%= link_to reimagine_root_url(path: "/hackathons"),
        class: "main-link",
        data: { role: "discover" } do %>
    Hackathons
  <% end %>
</li>
<li class="divider"></li>
<li>
  <%= link_to reimagine_root_url(path: "/software"),
        class: "main-link",
        data: { role: "projects" } do %>
    Projects
  <% end %>
</li>
<li class="divider"></li>
<li>

    <%= link_to reimagine_post_url,
                class: "main-link",
                data: { role: "for-orgs" } do %>
      Host a hackathon
    <% end %>
</li>`

can you use the syntax that is consistent with the root url? this is to sync staging

**Review by jchubacca — APPROVED** [review]

**Review by devpost-mzheng — APPROVED** [review]

**Pull request merged**